src/clients/users_auto.rb

```diff
@@ -36,8 +36,24 @@
 # This is a client for autoinstallation. It takes its arguments,
 # goes through the configuration and return the setting.
 # Does not do any changes to the configuration.
+
+Yast.import "Report"
+
 module Yast
   class UsersAutoClient < Client
+
+    # Checking double user entries
+    # (double username or UID)
+    # @param [Array] users to check
+    def check_users(users)
```

Comment:
If that method is not part of the public API (and it's not), I would move it to the end and set it as `private`.